### PR TITLE
Pia 4016 fix header layout for error and  no result screens

### PR DESCRIPTION
**Pull request by zladzeyka**
- Fix the layout on iPhone for error/ no result header
- Update text alignment and padding on iPad
- Refactor configuration of the constraints into view-related methods

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift [hunk 1]
```diff
@@ -166,23 +166,42 @@ class ErrorScreenViewController: UIViewController {
...
+                errorHeader.headerStack.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                errorContent.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                errorContent.widthAnchor.constraint(equalTo: view.widthAnchor,
+                                                    multiplier: Constants.iPadWidthMultiplier.rawValue)
+            ])
+        } else {
+            NSLayoutConstraint.activate([
+                errorHeader.headerStack.leadingAnchor.constraint(equalTo: view.leadingAnchor,
+                                                                 constant: Constants.iPadWidthPadding.rawValue),
+                errorHeader.headerStack.trailingAnchor.constraint(equalTo: view.trailingAnchor,
```
Comment (ValentinaIancu-Gini) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:189]: [clarification]: Also I've seen that in Figma the right margin of this `errorHeader` is 24 (`iPadWidthPadding` is 35).
Maybe that's why checking this on a device is not looking very good:
Screenshot 2023-06-23 at 13 12 33

In my opinion, there is too much space to the right where the content can fit but is not.

**Comment by ValentinaIancu-Gini** [comment]
Other than this, I've seen that the`errorHeader.headerStack` is getting out of the parent frame:
Screenshot 2023-06-23 at 13 19 30
Screenshot 2023-06-23 at 13 15 04
I didn't see any constraints on the top and bottom for this view.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift (hunk 1 above)
Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:189]: could you please add a link to Figma where you've seen 24?

Comment (ValentinaIancu-Gini) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:189]: Here. You need to select the text and press the option key with the mouse point on the right margin. 
https://www.figma.com/file/m4OtCvxWxPMeB0Z5k4Aeow/Gini-Photo-Payment-%2F-iOS?type=design&node-id=3991-53952&mode=design&t=TqYJ60xfD9Lf4pOU-4

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift [hunk 2]
```diff
@@ -310,5 +348,10 @@ final class NoResultScreenViewController: UIViewController {
         case twoButtonsHeight = 112
         case tableRowHeight = 44
         case sectionHeight = 70
+        case iPadWidthMultiplier = 0.7
```
[the remark this answers is not in the document]
Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift:351]: I will rewrite it.

**Review by ValentinaIancu-Gini — APPROVED** [review]
I've tested it on multiple devices and also with bold and larger text and it seems to work as expected.

**Pull request merged**